tests/frameworks/test_wsgi.py

```diff
@@ -3,383 +3,327 @@
 
 import time
 import urllib3
-import unittest
+import pytest
+from typing import Generator
 
-import tests.apps.flask_app
-from ..helpers import testenv
+from tests.apps import bottle_app
+from tests.helpers import testenv
 from instana.singletons import agent, tracer
+from instana.span.span import get_current_span
 
 
-class TestWSGI(unittest.TestCase):
-    def setUp(self):
+class TestWSGI:
+    @pytest.fixture(autouse=True)
+    def _setUp(self) -> Generator[None, None, None]:
```

Comment:
Please, use `resource` as the method name, as requested by you :-P

Reply:
Yes, please! Keeping a pattern will always be good.

Reply:
Done!